corehq/apps/app_manager/static/app_manager/js/app_diff_main.js

```diff
@@ -0,0 +1,4 @@
+hqDefine('app_manager/js/app_diff_main',[
```

Comment:
Since this only has one dependency, you can just use `app_manager/js/source_files` as the `requirejs_main` modules for `app_diff.html` and delete this file.